external/src/main/resources/application.properties

```diff
@@ -149,3 +151,8 @@ quarkus.http.non-application-root-path=/
 
 engine.triggers-lifecycle.disabled=true
 engine.actions-store.disabled=true
+
+# Enables LightweightEngine which is an engine implementation that does not rely on Infinispan or Drools.
+lightweight-engine.enabled=true
```

Comment:
This is enabled to show that test are passing with the new implementation. I will disable it before we merge the PR though.